CMakeLists.txt

```diff
@@ -14,9 +14,8 @@ cable_configure_toolchain(DEFAULT cxx11)
 set(HUNTER_CONFIGURATION_TYPES Release CACHE STRING "Build type of Hunter packages")
 set(HUNTER_JOBS_NUMBER 6 CACHE STRING "Number of parallel builds used by Hunter")
 HunterGate(
-    URL "https://github.com/ruslo/hunter/archive/v0.23.112.tar.gz"
-    SHA1 "4b894e1d5d203f0cc9a77431dbb1b486ab6f4430"
-    LOCAL
+    URL "https://github.com/cpp-pm/hunter/archive/v0.23.300.tar.gz"
```

Comment:
It is actually enough and easier to just replace the "ruslo" with "cpp-pm" and keep the same package version and SHA1. This saves troubles with required CMake version upgrade and adaptations to newer Boost version.